bower.json

```diff
@@ -9,14 +9,15 @@
   },
   "main": "dist/fxpay.min.js",
   "ignore": [
-    ".bowerrc",
-    ".gitignore",
-    ".jshintrc",
-    ".travis.yml",
+    "**/.*",
+    "Gruntfile.js",
     "build",
     "example",
-    "node_modules",
     "karma.conf.js",
+    "lib",
```

Comment:
I would say it makes more sense for them to use the built file. At least right now. Including all the pieces just means their build step e.g. r.js optimization would have more work to do. Also this means the lib is responsible for the distribution build which is almost certainly the better bet. 

Can always re-visit this in the future if we want to do something different.